src/main/resources/messages.properties

```diff
@@ -13,9 +13,15 @@ ExceedTotal.courseConfigurationForm.defaultMinutesPerSession = Must not exceed t
 typeMismatch.makeupForm.entries.minutesMadeUp = Must be a valid number.
 Min.makeupForm.entries.minutesMadeUp = Must be greater than 0.
 
+Required.courseConfigurationForm.entries.assignmentName = Assignment Name is required.
+Required.courseConfigurationForm.entries.assignmentPoints = Assignment Points is a required field and has to be greater than 0.
+Required.courseConfigurationForm.entries.statusPoints = All status point fields are required.
+Max.courseConfigurationForm.entries = Point values must be set between 0 and 100.
+
 typeMismatch.makeupForm.entries.dateMadeUp = Must be a valid Date dd/mm/yyyy
 
 typeMismatch.makeupForm.entries.dateOfClass = Must be a valid Date dd/mm/yyyy
 Required.makeupForm.entries.dateOfClass = This is required.
 
-Max.makeupForm.entries.projectDescription = Must be less than 256 characters.
\ No newline at end of file
+Max.makeupForm.entries.projectDescription = Must be less than 256 characters.
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove the whitespaces between the key and the separator. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.ome.ksu.edu/coding_rules#rule_key=jproperties%3Aseparator-convention)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove the whitespaces between the separator and the value. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.ome.ksu.edu/coding_rules#rule_key=jproperties%3Aseparator-convention)